athena-federation-sdk/athena-federation-sdk.yaml

```diff
@@ -10,7 +10,7 @@ Metadata:
     Labels:
       - athena-federation
     HomePageUrl: 'https://github.com/awslabs/aws-athena-query-federation'
-    SemanticVersion: 2020.5.1
+    SemanticVersion: 2020.17.1
```

Comment:
Line 50 should also be updated (CodeUri: "./target/aws-athena-federation-sdk-2020.5.1-withdep.jar").

I don't seem to be able to comment on unchanged sections of these files.

Reply:
good catch, updated